pkgs/top-level/python-packages.nix

```diff
@@ -4609,6 +4609,98 @@ in modules // {
     };
   };
 
+  distributed = buildPythonPackage rec {
+
+    name = "distributed-${version}";
+    version = "1.10.0";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/d/distributed/${name}.tar.gz";
+      sha256 = "11bp2rs52fhcqlgyrlh3cf31ck07mys38mrkf98vjl380lyjj357";
+    };
+
+    buildInputs = with self; [ pytest docutils ];
+    propagatedBuildInputs = with self; [
+      dask six boto3 s3fs tblib locket msgpack click cloudpickle tornado
+      psutil botocore
+    ] ++ (if !isPy3k then [ singledispatch ] else []);
+
+    # AttributeError: module 'pytest' has no attribute 'config'
+    doCheck = false;
```

Comment:
I tried the invocation they're using for travis but that still doesn't work `ValueError: no option named '--runslow'` 

This package also seems to need a successful install before running the tests (i.e. python setup.py install has to install binaries into some PATH accessible location).

I spent a fair while on trying to get tests to run but not sure it's easy.